packages/expo-notifications/src/ServerRegistrationModule.web.ts

```diff
@@ -26,7 +27,17 @@ export default {
 
     return installationId;
   },
+  getLastRegistrationInfoAsync: async () => {
+    return localStorage.getItem(LAST_REGISTRATION_INFO_KEY);
+  },
+  setLastRegistrationInfoAsync: async (lastRegistrationInfo: string | null) => {
+    if (lastRegistrationInfo) {
+      localStorage.setItem(LAST_REGISTRATION_INFO_KEY, lastRegistrationInfo);
+    } else {
+      localStorage.removeItem(LAST_REGISTRATION_INFO_KEY);
+    }
+  },
```

Comment:
Wrapped in try/catch and wrapped error into `CodedError`. All `set(Last)RegistrationInfoAsync` calls (well, one) are wrapped with another try/catch that prints a user-readable warning (more user-readable than "localStorage failed"):

https://github.com/expo/expo/blob/6f6e2018a9e53cf93c472f8ffdb0ff8afaa89a8c/packages/expo-notifications/src/DevicePushTokenAutoRegistration.fx.ts#L18-L26

https://github.com/expo/expo/blob/6f6e2018a9e53cf93c472f8ffdb0ff8afaa89a8c/packages/expo-notifications/src/getExpoPushTokenAsync.ts#L93-L106

Do you think it would make more sense not to display a warning / ignore `localStorage` set error?

Reply:
It seems useful to log it for now. Later I could see it being better to document the behavior and remove the extra logging.